terragrunt/modules/aws-organization/groups.tf

```diff
@@ -156,6 +163,62 @@ resource "aws_ssoadmin_permission_set_inline_policy" "triagebot_access" {
   permission_set_arn = aws_ssoadmin_permission_set.triagebot_access.arn
 }
 
+// Release team permission to start a new release
+
+resource "aws_ssoadmin_permission_set" "start_release" {
+  instance_arn = local.instance_arn
+  name         = "StartRelease"
+}
+
+resource "aws_ssoadmin_permission_set_inline_policy" "start_release" {
+  instance_arn       = local.instance_arn
+  permission_set_arn = aws_ssoadmin_permission_set.start_release.arn
+
+  inline_policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Effect   = "Allow"
+        Action   = "lambda:InvokeFunction"
+        Resource = "arn:aws:lambda:us-west-1:890664054962:function:start-release"
+      },
+      {
+        Effect = "Allow"
+        Action = [
+          // This must NOT include codebuild:StartBuild, starting builds must
```

Comment:
Should we explicitly deny it perhaps?

Reply:
Done. Also removed a permission that is not actually needed.